### PR TITLE
 Corrected-Grammar-consistency-about-verb

### PR DESCRIPTION
**Pull request by Ayush9026**
**Description** -  Grammar consistency about verb form in singular and plural, whether to add an "s", in feature-gates docs #43795

solved this issue added "s" where needed.

### CODE REVIEW

**Comment by PranitRout07** [comment]
I suggest to change defaulting to default here . It is at line number 504 in feature-gates.md file.
Screenshot 2023-11-03 182047

Change 'fine-tuing' to 'fine-tuning' at line number 614 and line number 615.
Screenshot 2023-11-03 182846

**Review by amolmote — COMMENTED** [review]
lgtm.

seems the change is necessary to have.

**Comment by amolmote** [comment]
fix the checks and review the whole file again, do the possible changes that you can think valid to have.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -375,9 +375,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 
 - `AdmissionWebhookMatchConditions`: Enable [match conditions](/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchconditions)
```
Comment (windsonsea) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:376]:
```suggestion
- `AdmissionWebhookMatchConditions`: Enables [match conditions](/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchconditions)
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -375,9 +375,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
   on mutating & validating admission webhooks.
-- `APIListChunking`: Enable the API clients to retrieve (`LIST` or `GET`)
+- `APIListChunking`: Enables the API clients to retrieve (`LIST` or `GET`)
   resources from API server in chunks.
-- `APIPriorityAndFairness`: Enable managing request concurrency with
+- `APIPriorityAndFairness`: Enables managing request concurrency with
   prioritization and fairness at each server. (Renamed from `RequestManagement`)
 - `APIResponseCompression`: Compress the API responses for `LIST` or `GET` requests.
 - `APISelfSubjectReview`: Activate the `SelfSubjectReview` API which allows users
```
Comment (windsonsea) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:383]:
```suggestion
- `APISelfSubjectReview`: Activates the `SelfSubjectReview` API which allows users
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -427,37 +427,37 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
   allowing you to scrape health check metrics.
 - `ConsistentHTTPGetHandlers`: Normalize HTTP get URL and Header passing for lifecycle
   handlers with probers.
-- `ConsistentListFromCache`: Allow the API server to serve consistent lists from cache.
+- `ConsistentListFromCache`: Allows the API server to serve consistent lists from cache.
 - `ContainerCheckpoint`: Enables the kubelet `checkpoint` API.
   See [Kubelet Checkpoint API](/docs/reference/node/kubelet-checkpoint-api/) for more details.
 - `ContextualLogging`: When you enable this feature gate, Kubernetes components that support
    contextual logging add extra detail to log output.
 - `CronJobsScheduledAnnotation`: Set the scheduled job time as an
```
Comment (windsonsea) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:448]:
```suggestion
- `CronJobsScheduledAnnotation`: Sets the scheduled job time as an
```

**Review by windsonsea — COMMENTED** [review]
Are the addition of 's' in these cases necessary as well?

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -427,37 +427,37 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `CloudDualStackNodeIPs`: Enables dual-stack `kubelet --node-ip` with external cloud providers.
   See [Configure IPv4/IPv6 dual-stack](/docs/concepts/services-networking/dual-stack/#configure-ipv4-ipv6-dual-stack)
   for more details.
-- `ClusterTrustBundle`: Enable ClusterTrustBundle objects and kubelet integration.
-- `ComponentSLIs`: Enable the `/metrics/slis` endpoint on Kubernetes components like
+- `ClusterTrustBundle`: Enables ClusterTrustBundle objects and kubelet integration.
+- `ComponentSLIs`: Enables the `/metrics/slis` endpoint on Kubernetes components like
   kubelet, kube-scheduler, kube-proxy, kube-controller-manager, cloud-controller-manager
   allowing you to scrape health check metrics.
 - `ConsistentHTTPGetHandlers`: Normalize HTTP get URL and Header passing for lifecycle
```
Comment (Ritikaa96) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:441]:
```suggestion
- `ConsistentHTTPGetHandlers`: Normalizes HTTP get URL and Header passing for lifecycle
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -427,37 +427,37 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 - `ContainerCheckpoint`: Enables the kubelet `checkpoint` API.
   See [Kubelet Checkpoint API](/docs/reference/node/kubelet-checkpoint-api/) for more details.
 - `ContextualLogging`: When you enable this feature gate, Kubernetes components that support
    contextual logging add extra detail to log output.
 - `CronJobsScheduledAnnotation`: Set the scheduled job time as an
   {{< glossary_tooltip text="annotation" term_id="annotation" >}} on Jobs that were created
   on behalf of a CronJob.
-- `CronJobTimeZone`: Allow the use of the `timeZone` optional field in [CronJobs](/docs/concepts/workloads/controllers/cron-jobs/)
+- `CronJobTimeZone`: Allows the use of the `timeZone` optional field in [CronJobs](/docs/concepts/workloads/controllers/cron-jobs/)
 - `CRDValidationRatcheting`: Enable updates to custom resources to contain
```
Comment (Ritikaa96) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:452]:
```suggestion
- `CRDValidationRatcheting`: Enables updates to custom resources to contain 
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 6]
```diff
@@ -485,7 +485,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   for more details.
 - `EfficientWatchResumption`: Allows for storage-originated bookmark (progress
   notify) events to be delivered to the users. This is only applied to watch operations.
-- `EventedPLEG`: Enable support for the kubelet to receive container life cycle events from the
+- `EventedPLEG`: Enablse support for the kubelet to receive container life cycle events from the
```
Comment (Ritikaa96) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:488]:
```suggestion
- `EventedPLEG`: Enables support for the kubelet to receive container life cycle events from the
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 7]
```diff
@@ -564,7 +564,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   If the `KMSv2` feature gate is not enabled in your cluster, the value of the `KMSv2KDF` feature gate has no effect.
 - `KubeProxyDrainingTerminatingNodes`: Implement connection draining for
```
Comment (Ritikaa96) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:565]:
```suggestion
- `KubeProxyDrainingTerminatingNodes`: Implements connection draining for
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 8]
```diff
@@ -581,9 +581,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `KubeletPodResources`: Enable the kubelet's pod resources gRPC endpoint. See
```
Comment (Ritikaa96) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:581]:
```suggestion
- `KubeletPodResources`: Enables the kubelet's pod resources gRPC endpoint. See
```

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 9]
```diff
@@ -720,46 +720,46 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+- `TopologyManagerPolicyBetaOptions`: Allows fine-tuning of topology manager policies,
   experimental, Beta-quality options.
   This feature gate guards *a group* of topology manager options whose quality level is beta.
   This feature gate will never graduate to stable.
-- `TopologyManagerPolicyOptions`: Allow fine-tuning of topology manager policies,
+- `TopologyManagerPolicyOptions`: Allows fine-tuning of topology manager policies,
 - `UnknownVersionInteroperabilityProxy`: Proxy resource requests to the correct peer kube-apiserver when
   multiple kube-apiservers exist at varied versions.
   See [Mixed version proxy](/docs/concepts/architecture/mixed-version-proxy/) for more information.
 - `UserNamespacesSupport`: Enable user namespace support for Pods.
```
Comment (Ritikaa96) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:756]:
```suggestion
- `UserNamespacesSupport`: Enables user namespace support for Pods.
```

**Review by Ritikaa96 — CHANGES_REQUESTED** [review]

**Comment by Ritikaa96** [comment]
Hi @Ayush9026 , there are more grammatical errors in this doc , can you again review and correct them:
For eg . 
`Feature gates are a set of **key=value** pairs that describe Kubernetes features`
should be
`Feature gates are a set of key-value pairs that describe Kubernetes features`

**Comment by Ritikaa96** [comment]
Hi @Ayush9026 Please resolve the conflicts.

**Comment by natalisucks** [comment]
@Ayush9026 Hi there! With the inactivity on this PR when it comes to following up on feedback, I'll be closing it. However, please feel free to come back and open this PR when you have the capacity to work on it once again. Thank you!
/close

**Comment by Ayush9026** [comment]
/ open

**Pull request closed without merging**